scripts/update-version.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Check if argument is provided
+if [ $# -eq 0 ]; then
+    echo "Usage: $0 <input>"
+    exit 1
+fi
+
+VERSION="$1"
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+# Validate the version format (e.g., 1.0.0 or 1.0.0-rc.1)
+if ! [[ "$VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z0-9]+(\.[0-9]+)*)?$ ]]; then
+    echo "Error: Invalid version format. Use 'X.Y.Z' or 'X.Y.Z-rc.1'."
+    exit 1
+fi
+
+echo "Updating MapboxMaps Flutter SDK version: $VERSION"
+
+PUBSPEC_FILE="$SCRIPT_DIR/../pubspec.yaml"
+sed -i '' "s/^version: .*/version: $VERSION/" "$PUBSPEC_FILE"
+
+PODSPEC_FILE="$SCRIPT_DIR/../ios/mapbox_maps_flutter.podspec"
+sed -i '' "s/\(s\.version *= *\)\'[^\']*\'/\1\'$VERSION\'/" "$PODSPEC_FILE"
+
```

Comment:
I think Android's `build.gradle` should be updated as well

Reply:
this is not yet updating the dependency version, only to update the version of the package. I am working on the dep version sync/update that will update build.gradle :-D

Reply:
Ah, sorry, misunderstood the script, looking good then 👍